cognite/client/_api/three_d.py

```diff
@@ -217,6 +229,8 @@ def update(
                 >>> res = client.three_d.models.update(my_update)
 
         """
+        # Note that we cannot use the ThreeDModelWrite to update as the write format of a 3D model
+        # does not have ID or External ID, thus no identifier to know which model to update.
```

Comment:
This is awkward... imo, there's no excuse for external ID to be absent 😢 Should we raise? Or is the API message clear?

Reply:
It is actually an ValueError and it complains that there is no ID, External ID or Space to identify, so I think it is clear.